From f4e4e7d82be37c829198a3bdf33693b4212eee15 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 6 Oct 2018 21:37:33 +0300
Subject: [PATCH 12/22] kernel: Correct CROSS_COMPILE_ARM32 toolchain

* Missed in the original commit that switched to
  androidkernel toochain

Change-Id: I9b6d2e7216b48d22bcb7eec62c4fd9c05a895d18
---
 build/tasks/kernel.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/tasks/kernel.mk b/build/tasks/kernel.mk
index ec26deda..3555d256 100644
--- a/build/tasks/kernel.mk
+++ b/build/tasks/kernel.mk
@@ -272,7 +272,7 @@ endif
 
 # Needed for CONFIG_COMPAT_VDSO, safe to set for all arm64 builds
 ifeq ($(KERNEL_ARCH),arm64)
-   KERNEL_CROSS_COMPILE += CROSS_COMPILE_ARM32="arm-linux-androideabi-"
+   KERNEL_CROSS_COMPILE += CROSS_COMPILE_ARM32="arm-linux-androidkernel-"
 endif
 
 ccache =
-- 
2.17.1

